include/mbgl/shaders/shader_defines.hpp

```diff
@@ -40,5 +40,73 @@ static constexpr auto maxUBOCountPerShader = std::max({static_cast<size_t>(backg
                                                        static_cast<size_t>(rasterUBOCount),
                                                        static_cast<size_t>(symbolUBOCount)});
 
+enum {
```

Comment:
I think it would be better to put these in their own file, because unlike the above they don't require a ton of includes.

Reply:
Let's postpone this split for my next PR: https://github.com/alexcristici/maplibre-native/pull/14/files, which is on top of this. Splitting the file now will cause some conflicts.

Reply:
Alright!